### ISSUE TITLE
Support OPDS

### ISSUE THREAD

**Issue by HLFH** [kobolabs/Kobo-Reader#24]
Hi,

Please support OPDS.
https://en.wikipedia.org/wiki/Open_Publication_Distribution_System

OPDS is the best way for an interconnected E-Reader with its own servers, its own Small Data, its own Personal Data, its own Ebooks...

Thanks in advance,

**Comment by tmortagne** [comment]
+1

That would indeed be a great feature.

**Comment by bartart3d** [comment]
+1

**Comment by tarioch** [comment]
+1

**Comment by nitmir** [comment]
A must have to allow me to easily transfer news to my ereader without having to plug it into my computer.

**Comment by HLFH** [comment]
@gtalusan Hi ! ;) Any news?

**Comment by gtalusan** [comment]
It's on my radar at the least, but it's not me that needs convincing that this is a cool feature.

**Comment by bkqc** [comment]
Who is it then?

**Comment by cadeon** [comment]
Me Too.

**Comment by bmintz** [comment]
If you agree, please click the +:smile: button at the top right of @HLFH's post, instead of commenting "+1" which adds nothing to the discussion.

**Comment by polytan02** [comment]
It would be great to have this.
Also, a web browser with login capabilities would be appreciated.

**Comment by silverhook** [comment]
@gtalusan 
> It's on my radar at the least, but it's not me that needs convincing that this is a cool feature.

So, who do we need to convince this is a good idea and where do we find them? ;)

**Comment by flowgunso** [comment]
@gtalusan This feature would make me use my Kobo again.

**Comment by lemmy04** [comment]
+99

**Comment by lemmy04** [comment]
by the way, there is a drop in app for nextcloud that lets you export a folder in your cloud as opds catalog.

**Comment by HLFH** [comment]
@lemmy04 Yes when Kobo-Reader will support OPDS, he will be able to interact with `files_opds` [plugin of Nextcloud](https://github.com/Yetangitu/owncloud-apps).

**Comment by lemmy04** [comment]
this feels quite ignored.

**Comment by ErikLentz** [comment]
I'm shopping for a new ereader and OPDS support is something I really want. If Kobo had it, I would have no other choice but to buy Kobo.

**Comment by theAlevan** [comment]
+1

**Comment by GianniBYoung** [comment]
Adding my voice, I would like to see this as a feature

**Comment by MaxHuiskes** [comment]
I would love this as a feature

**Comment by aacebedo** [comment]
> After 12 years, I sincerly doubt we will ever see this...

That's a shame as it seems to be a basic feature nowadays

**Comment by silverhook** [comment]
As @HLFH suggested, I also worked around this issue by simply using KOReader instead of the official software. It’s honestly great!
